tests/HTTP/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ set (HTTP_SRCS
 	${CMAKE_SOURCE_DIR}/src/HTTP/TransferEncodingParser.cpp
 	${CMAKE_SOURCE_DIR}/src/HTTP/UrlClient.cpp
 	${CMAKE_SOURCE_DIR}/src/HTTP/UrlParser.cpp
+	${CMAKE_SOURCE_DIR}/src/OSSupport/StackTrace.cpp
```

Comment:
I don't get, why this is necessary for this PR.

Reply:
Because `UNREACHABLE` uses `PrintStackTrace()` and so needs to link to it.